core/01_core.js

```diff
@@ -804,6 +804,8 @@
         typeof arg === "number" || arg === null || arg === undefined
       ) {
         return arg;
+      } else if (arg instanceof Uint8Array) {
+        `Uint8Array(${arg.length}) [${arg.slice(0, 10).join(", ")}]`;
```

Comment:
```suggestion
      } else if (TypedArrayPrototypeSymbolToStringTag(arg) === "Uint8Array") {
        `Uint8Array(${TypedArrayPrototypeGetLength(arg)}) [${
          TypedArrayPrototypeJoin(TypedArrayPrototypeSlice(arg, 0, 10), ", ")
        }]`;
```

Reply:
Thanks, I remember to use primordials - I just opened a WIP PR so I don't forget I was to update the buffer display.